tests/unit/modules/network/ios/test_ios_vlans.py

```diff
@@ -372,12 +483,42 @@ def test_ios_vlans_overridden_idempotent(self):
         )
         self.execute_module(changed=False, commands=[], sort=True)
 
-    def test_ios_delete_vlans_config(self):
+    def test_ios_delete_vlans(self):
         set_module_args(dict(config=[dict(vlan_id=150)], state="deleted"))
         result = self.execute_module(changed=True)
         commands = ["no vlan 150"]
         self.assertEqual(result["commands"], commands)
 
+    def test_ios_delete_vlans_config(self):
```

Comment:
ref https://github.com/ansible-collections/cisco.ios/actions/runs/6429392546/job/17460328093?pr=917#step:10:793 unit test failure

Reply:
Thanks for the review, Sagar. Fixed.